### PR TITLE
Fixing issue : 113.

### PR DESCRIPTION
**Pull request by bailaC**
This will fix https://github.com/ros-controls/ros2_controllers/issues/113.

### CODE REVIEW

**Comment by bailaC** [comment]
@destogl , @bmagyar , please have a look.

File: effort_controllers/test/test_load_joint_group_effort_controller.cpp [hunk 1]
```diff
@@ -29,7 +29,7 @@ TEST(TestLoadJointGroupVelocityController, load_controller)
 
   controller_manager::ControllerManager cm(
     std::make_unique<hardware_interface::ResourceManager>(
-      ros2_control_test_assets::minimal_robot_urdf),
+      ros2_control_test_assets::diffbot_urdf),
```
[the remark this answers is not in the document]
Comment (bailaC) [reply effort_controllers/test/test_load_joint_group_effort_controller.cpp:32]: This change is to pick the newly added 'diffbot_urdf' value from 'ros2_control_test_assets'.

[the remark this answers is not in the document]
Comment (destogl) [reply effort_controllers/test/test_load_joint_group_effort_controller.cpp:32]: Yeah, but why in "joint_group_effort_controller"? This is wrong controller, this should be only changed in the "DiffDriveController"

[the remark this answers is not in the document]
Comment (destogl) [reply effort_controllers/test/test_load_joint_group_effort_controller.cpp]: @bailaC  I correct it!

Please also take care in the future to run pre-commit to get format right the next time.

[the remark this answers is not in the document]
Comment (bailaC) [reply effort_controllers/test/test_load_joint_group_effort_controller.cpp]: Thank you @destogl

**Pull request merged**